include/ghex/transport_layer/ucx/context.hpp

```diff
@@ -87,6 +87,7 @@ namespace gridtools {
             private: // members
                 
                 thread_primitives_type&    m_thread_primitives;
+                mpi::hwtopo&               m_hwtopo;
```

Comment:
Is `hwtopo` modifiable? If not maybe this can be turned into
```suggestion
                mpi::hwtopo const&               m_hwtopo;
```

Reply:
sure